website/docs/r/apigatewayv2_api.html.markdown

```diff
@@ -52,8 +52,11 @@ Defaults to `$request.method $request.path`.
 * `target` - (Optional) Part of _quick create_. Quick create produces an API with an integration, a default catch-all route, and a default stage which is configured to automatically deploy changes.
 For HTTP integrations, specify a fully qualified URL. For Lambda integrations, specify a function ARN.
 The type of the integration will be `HTTP_PROXY` or `AWS_PROXY`, respectively. Applicable for HTTP APIs.
+* `body` - (Optional) An OpenAPI specification that defines the set of routes and integrations to create as part of the HTTP APIs. Supported only for HTTP APIs.
 * `version` - (Optional) A version identifier for the API.
 
+__Note__: If the `body` argument is provided, the `info` field will also be used to update the attributes of the API Gateway by AWS. It's a good practice to keep them with the same values. However, to avoid further issues, We will not respect those definitions in your `info` field intentionally.
```

Comment:
Could you clarify what you mean by `info` field here?

Also, could you add a note similar to the REST API equivalent

```
__Note__: If the `body` argument is provided, the OpenAPI specification will be used to configure the resources, methods and integrations for the Rest API. If this argument is provided, the following resources should not be managed as separate ones, as updates may cause manual resource updates to be overwritten:

* `aws_api_gateway_resource`
* `aws_api_gateway_method`
* `aws_api_gateway_method_response`
* `aws_api_gateway_method_settings`
* `aws_api_gateway_integration`
* `aws_api_gateway_integration_response`
* `aws_api_gateway_gateway_response`
* `aws_api_gateway_model`
```

about which other resources are implicitly created and that they _shouldn't_ be `terraform import`ed as they are managed via the API spec?
I think at least `aws_apigatewayv2_route` and `aws_apigatewayv2_integration` get implicitly created.

Reply:
Sure, I will add it this weekend.